api/index.html

```diff
@@ -276,16 +276,16 @@ <h3 style="margin: 10px auto 0px;width:100%;font-size:14pt;font-weight:normal">V
     <h5>Setup</h5>
 
     <p>
-      Download <a href="https://cdn.jsdelivr.net/npm/nexusui@2.0.13/dist/NexusUI.js" download>NexusUI.js</a>. <br />
+      Download <a href="https://cdn.jsdelivr.net/npm/nexusui@2.1.5/dist/NexusUI.js" download>NexusUI.js</a>. <br />
```

Comment:
Possibly use "@next" here so this page never goes out of sync with the latest NPM version number?